pkg/machine/machinesetlist.go

```diff
@@ -0,0 +1,54 @@
+package machine
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ListWorkerMachineSets returns the list of slice of SetBuilder objects in a namespace on a cluster.
```

Comment:
"the list of slice" - is that right?

Reply:
Good catch, should be "returns a slice of SetBuilder objects".  Will fix and push it.  thank you !